File: 1.FDP/5°Competencia/Java/Funciones/Ejercicio1.java
import java.util.Scanner;	

public class Ejercicio1 {

// Funcion--------------------------------------------------------------------------------------------
	public static double CC (double P, double T){
		return P - T;
	}
//----------------------------------------------------------------------------------------------------

	public static void main(String[] args){
		Scanner scan= new Scanner(System.in);
		System.out.println("Calcular el cambio de una compra.\n");
		double matriz[][] = new double[3][3];
		
		// Procedimiento.----------------------------------------------------------------------------
		System.out.println("El programa se repetiira 3 veces");
		// Datos-------------------------------------------------------------------------------------
		for (int i=0; i<3; i++) {
			System.out.println("Ingresar el total de compra:");
			matriz[i][0] = scan.nextDouble();
			if (matriz[i][0] > 0) {
				System.out.println("Ingresar la cantidad de pago:");
				matriz[i][1]= scan.nextDouble();
				if (matriz[i][1] >= matriz[i][0]) {
					matriz[i][2]= CC(matriz[i][1], matriz[i][0]);
					System.out.println("//////////////////////////");
				}else{
					System.out.println("Error en el pago");
					System.out.println("//////////////////////////");
				}
			}else{
				System.out.println("Error en el total.\n");
				System.out.println("//////////////////////////");
			}
		}
		//-----------------------------------------------------------------------------------------

		// Impresion de resultados-----------------------------------------------------------------
		for (int i = 0; i < 3; i++) {
			if( matriz[i][1] > 0 && matriz[i][1] >= matriz[i][0] ){
			System.out.println ("Ejercicios " + (i + 1) );
			System.out.println("Cambio de $" + matriz[i][0] +
					" - $" + matriz[i][1] + "= $" + matriz[i][2] + "\n");
			}
		}
		scan.close();
		//----------------------------------------------------------------------------------------

		// Creador y fecha------------------------------------------------------------------------
		System.out.println("\n");
		System.out.println("Creador: Jesus A. Mex Damian 21/11/2022");
		//----------------------------------------------------------------------------------------
	}
}

File: 1.FDP/5°Competencia/C++/Funciones/Ejercicio1.cpp
#include <iostream>
#include <cmath>
using namespace std;

// Funcion---------------------------------------------------------------------------------
double Cambio (double T, double P){
	return P - T;
}
//-----------------------------------------------------------------------------------------

int main(){
	cout << "Calcular el cambio de una compra.\n";
	double matriz[1000][3] ={0};
	int respuesta; 
	int i = 1;
	
	// Procedimiento.----------------------------------------------------------------------------
	cout << "Cuantas veces deseas que se repita (Hasta 1000 veces): ";
	cin >> respuesta;
	// Datos-------------------------------------------------------------------------------------
	for ( int i=0; i<respuesta; i++) {
		cout << "Ingresar el total:$";
		cin >>matriz[i][0];
		if (matriz[i][0] > 0){
			cout << "Ingresar el pago:$";
			cin >> matriz[i][1];
			if (matriz[i][1] >= matriz[i][0] ) {
	// Formula-----------------------------------------------------------------------------------
				matriz[i][2]= Cambio (matriz[i][0], matriz[i][1]);
				cout << "//////////////////////////////////.\n";
			} else{ 
			cout << "Error en el pago.\n";
			cout << "///////////////////////.\n";
			}
		} else {
			cout << "Error en el total.\n";
			cout << "///////////////////////.\n";
		}
	}
	//-----------------------------------------------------------------------------------------

	// Impresion de resultados-----------------------------------------------------------------
	for (int i=0;i<respuesta; i++){
		if( matriz[i][1] > 0 && matriz[i][1] >= matriz[i][0] ){
		cout << "Ejercicios " << i + 1 << "\n";
		cout << "Cambio de $" << matriz[i][1] << " - $" 
		<< matriz[i][0] << "= $" << matriz[i][2] << "\n";
		}
	}
	//------------------------------------------------------------------------------------------

	// Creador y fecha------------------------------------------------------------------------
	cout <<"\n";
	cout <<"Creador: Jesus A. Mex Damian 21/11/2022.";
	//-----------------------------------------------------------------------------------------

	return 0;
}